src/simpleModels/ServiceAddress.php

```diff
@@ -22,7 +22,8 @@ class ServiceAddress {
         "AddressLine2" => array("type" => "string"),
         "PlusFour" => array("type" => "string"),
         "AddressType" => array("type" => "string"),
-        "UnparsedAddress" => array("type" => "string") // Note that UnparsedAddress is Read-Only!
+        // Note that UnparsedAddress is Read-Only.
```

Comment:
@brianluisgomez How about expanding the comment to also indicate passing it does not automatically populate individual Address fields? `read-only` generally means once it's set it cannot be updated!

Reply:
Sounds good, I expanded the comment